### PR TITLE
Template send

### PR DESCRIPTION
**Pull request by vaibhavppathak**

### CODE REVIEW

File: src/controllers/imap.js [hunk 1]
```diff
@@ -9,91 +9,99 @@ export class ImapController extends BaseAPIController {
...
-                } else {
-                    res.json({ status: 0, message: "error", data: "email not found in database" });
-                }
-            })
-            .catch(this.handleErrorResponse.bind(null, res));
+      statusActive = (req, res) => {
+          this._db.Imap.findOne({ where: { email: req.params.email } })
+              .then((result) => {
+                  if (result) {
+                      var imap = new Imap({
```
Comment (mageexxcel) [inline src/controllers/imap.js]: move this to service. imap connection. this has become very long

File: src/controllers/imap.js [hunk 2]
```diff
@@ -9,91 +9,99 @@ export class ImapController extends BaseAPIController {
...
+                          password: result.password,
+                          host: result.imap_server,
+                          port: result.server_port,
+                          tls: result.type
+                      });
+                      imap_connection(imap, (err) => {
+                          if (err) {
+                              res.json({ status: 0, message: "error", data: err });
+                          } else {
+                              this._db.Imap.update({ status: "FALSE" }, { where: { email: req.body.email } })
```
Comment (mageexxcel) [inline src/controllers/imap.js]: wrong status: "FALSE"

File: src/controllers/inbox.js [hunk 3]
```diff
@@ -1,28 +1,52 @@
...
+                res.json({
+                    data
+                });
+            }
+        });
+    }
+
+    /* Get Emails By EmailId*/
+    getByEmailId = (req, res, next) => {
+        console.log(req.params.emailid)
```
Comment (mageexxcel) [inline src/controllers/inbox.js]: ?

Comment (vaibhavppathak) [reply src/controllers/inbox.js]: done

**Pull request merged**